devices.js

```diff
@@ -744,7 +744,13 @@ const devices = [
                 (cb) => device.bind('msTemperatureMeasurement', coordinator, cb),
                 (cb) => device.bind('msOccupancySensing', coordinator, cb),
                 (cb) => device.report('genPowerCfg', 'batteryPercentageRemaining', 0, 1000, 0, cb),
-                (cb) => device.report('msOccupancySensing', 'occupancy', 0, 600, null, cb),
+                // Device should report at least short before occupancyTimeout.
+                // Warning: in case options occupancy_timeout is used instead of default
+                // occupancyTimeout, max reporting intervall needs to be set to an interval
+                // shorter than occupancy_timeout. Else our timer may timeout before
+                // device reports its status. (important in case of continous motion)
+                (cb) => device.report('msOccupancySensing', 'occupancy',
+                    0, Math.round(fz.occupancyTimeout * 0.9), null, cb),
```

Comment:
AFAIK the occupancyTimeout is not exported, how can this work?

Reply:
done